src/status_im2/subs/contact.cljs

```diff
@@ -68,6 +68,21 @@
  (fn [contacts]
    (contact.db/get-active-contacts contacts)))
 
+(re-frame/reg-sub
+ :contacts/active-sections
```

Comment:
I think we should try to always have unit tests for layer-3 subscriptions. It's cheap/easy to write and valuable. I understand it's just a refactoring, but given you put the effort into extracting them to subscriptions, the [human/boy/girl/etc scout rule](https://dev.to/ben/the-boy-scout-rule-is-now-the-scout-rule-420g) can be applied here.

One way we can increase the quality of our app without relying on slow e2e tests is to thoroughly test anything that touches the app db (layer-3 subs, event handlers, effects, etc).